### PR TITLE
Node template love

### PR DESCRIPTION
**Pull request by danforbes**
Better description in node/Cargo.toml
Example dispatchables should have non-zero weights
Changes to README to bring in line with current DevHub Node Template

### CODE REVIEW

File: bin/node-template/pallets/template/src/lib.rs [hunk 1]
```diff
@@ -75,7 +75,7 @@ decl_module! {
 		/// Just a dummy entry point.
 		/// function that can be called by the external world as an extrinsics call
 		/// takes a parameter of the type `AccountId`, stores it, and emits an event
-		#[weight = 0]
+		#[weight = 10_000]
```
Comment (riusricardo) [inline bin/node-template/pallets/template/src/lib.rs:78]: I'm not super familiar with this but Shawn could suggest another way to calculate a simple default value.
Something like:  `(10_000 + T::DbWeight::get().writes(1))]`

File: bin/node-template/pallets/template/src/lib.rs [hunk 2]
```diff
@@ -91,7 +91,7 @@ decl_module! {
 
 		/// Another dummy entry point.
 		/// takes no parameters, attempts to increment storage value, and possibly throws an error
-		#[weight = 0]
+		#[weight = 10_000]
```
Comment (riusricardo) [inline bin/node-template/pallets/template/src/lib.rs:94]: Same as above `#[weight = (10_000 + T::DbWeight::get().reads(1))]`?

**Comment by bkchr** [comment]
Update?

**Comment by danforbes** [comment]
Converted this to a draft because I need to make a few more small changes but I will get this taken care of today.

File: bin/node-template/README.md [hunk 3]
```diff
@@ -1,54 +1,76 @@
...
 ```
 
-Build Wasm and native code:
+### Manual Setup
+
+Find manual setup instructions at the [Substrate Developer Hub](https://substrate.dev/docs/en/knowledgebase/getting-started/#manual-installation).
+
+### Build
+
+Once the development environment is setup, build the node template. This command will build the
```
Comment (joepetrowski) [inline bin/node-template/README.md:26]:
```suggestion
Once the development environment is set up, build the node template. This command will build the
```

Comment (danforbes) [reply bin/node-template/README.md:26]: Should we change this on line 20 too then?

Comment (danforbes) [reply bin/node-template/README.md:26]: I know I see

**Comment by danforbes** [comment]
@bkchr I think this should be ready to merge now based on Ricardo's technical contributions/approval and the contributions/approval from Addie and Joe to the verbiage :pray: Let me know if there's anything I need to do to get CI or any kind of companion build passing :heavy_check_mark:

**Comment by bkchr** [comment]
> @bkchr I think this should be ready to merge now based on Ricardo's technical contributions/approval and the contributions/approval from Addie and Joe to the verbiage pray Let me know if there's anything I need to do to get CI or any kind of companion build passing heavy_check_mark

It does not compile :D

**Comment by danforbes** [comment]
LOL yes I just noticed that...looking into it now :facepalm:

**Comment by danforbes** [comment]
All set @bkchr. It looks like GitHub will allow me to merge this PR, but I'll leave it to the professionals :sweat_smile:

**Comment by riusricardo** [comment]
> LOL yes I just noticed that...looking into it now :facepalm: 
You  modified my commit :facepalm: . It was building, it had the Get trait

**Pull request merged**